package.json

```diff
@@ -45,7 +45,7 @@
   "devDependencies": {
     "ansi-magenta": "^0.1.1",
     "benchmarked": "^2.0.0",
-    "coffee-script": "^1.12.7",
+    "coffeescript": "^2.0.3",
```

Comment:
please set this as version `^1.12.7` since the newer version of `coffeescript` is failing on older versions of node.